forui/lib/src/widgets/tabs/tabs.dart

```diff
@@ -0,0 +1,168 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import 'package:flutter_localizations/flutter_localizations.dart';
+
+import 'package:forui/forui.dart';
+
+part 'tabs_style.dart';
+
+part 'tab_controller.dart';
+
+/// An object that represents a tab entry in a group of tabs.
+class FTabEntry {
+  /// The label.
+  final String? label;
+
+  /// A raw label.
+  final Widget? rawLabel;
+
+  /// The content of a tab.
+  final Widget content;
+
+  /// Creates a [FTabs].
+  FTabEntry({
+    required this.content,
+    this.label,
+    this.rawLabel,
+  }) : assert(label == null || rawLabel == null,
+            'Cannot provide both a label and a rawLabel.');
```

Comment:
Please add unit tests to check the assertion. You can reference the tests for other widgets.